### DISCUSSION TITLE
cisTarget databases related papers

### DISCUSSION THREAD

**Discussion by MarcoMatta** [aertslab/SCENIC#267] [* General SCENIC questions]
Hi,
where Can I find related paper or explanation of cisTarget databases?

https://resources.aertslab.org/cistarget/

thanks

**Comment by ghuls** [comment]
PWM-based whole-genome rankings across species (gene-based): https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003731

Region-scoring and ranking: https://academic.oup.com/nar/article/43/W1/W57/2467895
